src/compiler/diagnosticMessages.json

```diff
@@ -6245,7 +6273,30 @@
         "category": "Message",
         "code": 95159
     },
-
+    "Add 'override' modifier": {
+        "category": "Message",
+        "code": 95160
+    },
+    "Remove 'override' modifier": {
+        "category": "Message",
+        "code": 95161
+    },
+    "Add all 'override' modifier": {
+        "category": "Message",
+        "code": 95162
+    },
+    "Remove all 'override' modifier": {
+        "category": "Message",
+        "code": 95163
+    },
+    "Convert to property declaration and add 'override' modifier": {
+        "category": "Message",
+        "code": 95164
+    },
+    "Convert all to property declaration and add 'override' modifier": {

```

Comment:
Unclear if we really need this quick fix.

Reply:
I think we needn't. `Convert all to property declaration` has been denied.